elmslie-android/src/main/java/vivid/money/elmslie/android/processdeath/StopElmOnProcessDeathController.kt

```diff
@@ -0,0 +1,6 @@
+package vivid.money.elmslie.android.processdeath
+
+interface StopElmOnProcessDeathController {
+
+    val isAllowedToRunMvi: Boolean
```

Comment:
Now this sound like mvi is disabled completely (which it does, but it's incorrect). Also, this library uses Elm, not mvi, so it would be better to use elm in the name. I suggest to rename it something reflecting process death handling.

Reply:
Maybe it should be renamed to something like isElmWillBeRecycled or isElmWillBeRecycledAfterProcessDeath